app/kuma-dp/cmd/context.go

```diff
@@ -56,8 +55,7 @@ func DefaultRootContext() *RootContext {
 	return &RootContext{
 		ComponentManager: component.NewManager(leader_memory.NewNeverLeaderElector()),
 		BootstrapGenerator: envoy.NewRemoteBootstrapGenerator(&http.Client{
-			Timeout:   10 * time.Second,
-			Transport: &http.Transport{TLSClientConfig: &tls.Config{InsecureSkipVerify: true}},
```

Comment:
doesn't this have wider implications?

Reply:
This is to download arbitrary resources by doings: `kumactl -f https://...` I think using InsecureSkipVersion is super dangerous here.